### PR TITLE
feat(cart,product): add introspection for Magento driver apollo tests

### PR DESCRIPTION
**Pull request by lderrickable**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe: testing
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/product/src/drivers/magento/models/bundled-product.ts [hunk 1]
```diff
@@ -13,10 +13,6 @@ export enum MagentoPriceTypeEnum {
 }
 
 export interface MagentoBundledProduct extends MagentoProduct {
```
Comment (damienwebdev) [inline libs/product/src/drivers/magento/models/bundled-product.ts:15]: Let's remove this from this PR, its outside scope.

File: libs/cart/src/drivers/magento/queries/fragments/cart-item.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const cartItemFragment = gql`
   fragment cartItem on CartItemInterface {
     id
     product {
-      ...product
+      ...magentoProduct
```
Comment (damienwebdev) [inline libs/cart/src/drivers/magento/queries/fragments/cart-item.ts:11]: Let's remove this from this PR, its outside scope.

File: libs/product/src/drivers/magento/queries/fragments/bundled-product.ts [hunk 3]
```diff
@@ -1,11 +1,7 @@
 import gql from 'graphql-tag';
 
-export const bundledProductFragment = gql`
-  fragment BundledProduct on BundleProduct {
-		dynamic_price
-		dynamic_sku
-		dynamic_weight
-		ship_bundle_items
+export const magentoBundledProductFragment = gql`
+  fragment magentoBundledProduct on BundleProduct {
```
Comment (damienwebdev) [inline libs/product/src/drivers/magento/queries/fragments/bundled-product.ts:4]: Let's remove this from this PR, its outside scope.

File: libs/product/testing/src/factories/magento/simple/simple.factory.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
```
Comment (damienwebdev) [inline libs/product/testing/src/factories/magento/simple/simple.factory.ts:1]: Let's remove this from this PR, its outside scope.

Comment (lderrickable) [reply libs/product/testing/src/factories/magento/simple/simple.factory.ts]: removed

File: libs/product/src/drivers/magento/queries/fragments/bundled-product.ts (hunk 3 above)
Comment (lderrickable) [reply libs/product/src/drivers/magento/queries/fragments/bundled-product.ts]: removed

File: libs/cart/src/drivers/magento/queries/fragments/cart-item.ts (hunk 2 above)
Comment (lderrickable) [reply libs/cart/src/drivers/magento/queries/fragments/cart-item.ts]: removed

File: libs/product/src/drivers/magento/models/bundled-product.ts (hunk 1 above)
Comment (lderrickable) [reply libs/product/src/drivers/magento/models/bundled-product.ts]: reverted

**Comment by lderrickable** [comment]
should be good

File: libs/cart/package.json [hunk 5]
```diff
@@ -34,8 +34,9 @@
     "angular-in-memory-web-api": "0.0.0-PLACEHOLDER",
     "apollo-angular": "0.0.0-PLACEHOLDER",
     "apollo-client": "0.0.0-PLACEHOLDER",
-    "graphql-tag": "0.0.0-PLACEHOLDER"
-  },
+    "graphql-tag": "0.0.0-PLACEHOLDER",
+		"@daffodil/driver": "0.0.0-PLACEHOLDER"
+	},
   "devDependencies": {
```
Comment (damienwebdev) [inline libs/cart/package.json:40]: For lerna, `@daffodil/driver` needs to be in the `devDependencies` as well.

Comment (lderrickable) [reply libs/cart/package.json:40]: done

**Pull request merged**